README.md

```diff
@@ -1,20 +1,60 @@
-# Tripper
+Tripper
+=======
+_Triplestore wrapper for Python providing a simple and consistent interface to a range of triplestore backends._
```

Comment:
```suggestion
# Tripper

_A triplestore wrapper for Python - the best ride when handling any triplestore._
```

Reply:
We should sit down and agree on a common way to do things. 

The README file is a text file intended to be read by humans, hence I think we should use the notation that is most natural.

For me

    Main header
    ===========

is much more natural than

    # Main header

which looks more like a comment to me...

Reply:
I see no problem with mixing style as long as it makes the document more readable.

I agree that it is not obvious that a tripple-hash notation is a sub-sub-header (but there is nothing we can do with that). At the same time a tripple-hash stands out from the running text (much more than a single-hash) and gives a natural sectioning feeling for the eye, so it is not so bad either.

Reply:
> I see no problem with mixing style as long as it makes the document more readable.

This makes things very confusing and unintuitive in my opinion. And general bad practice.

> I agree that it is not obvious that a tripple-hash notation is a sub-sub-header (but there is nothing we can do with that). (...)

Well, we could make the first and second header be hashes as well, then it would be obvious ...

Reply:
I think we have to discuss and agree on a style to use. No reason to spend more time arguing back and forth (although it might refine our arguments...)